From c25a0e0dfa0da75bf9643c7a67ec60908b185288 Mon Sep 17 00:00:00 2001
From: Youri Querry <youri.querry_1@nxp.com>
Date: Thu, 7 Mar 2019 19:18:33 -0500
Subject: [PATCH 482/741] soc: fsl: dpio: Increase timeout for QBMan Management
 Commands

commit bef16ab4219cd27d81b03ff5760d570d49924387 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

The timeout for QBMan Management Commands can falsely trigger on a busy
system. This patch doubles the timeout to avoid the false error reports

Signed-off-by: Youri Querry <youri.querry_1@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/soc/fsl/dpio/qbman-portal.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/soc/fsl/dpio/qbman-portal.h b/drivers/soc/fsl/dpio/qbman-portal.h
index d501a2966f64..c35e53ef5394 100644
--- a/drivers/soc/fsl/dpio/qbman-portal.h
+++ b/drivers/soc/fsl/dpio/qbman-portal.h
@@ -1,7 +1,7 @@
 /* SPDX-License-Identifier: (GPL-2.0+ OR BSD-3-Clause) */
 /*
  * Copyright (C) 2014-2016 Freescale Semiconductor, Inc.
- * Copyright 2016 NXP
+ * Copyright 2016-2019 NXP
  *
  */
 #ifndef __FSL_QBMAN_PORTAL_H
@@ -445,7 +445,7 @@ static inline int qbman_swp_CDAN_set_context_enable(struct qbman_swp *s,
 static inline void *qbman_swp_mc_complete(struct qbman_swp *swp, void *cmd,
 					  u8 cmd_verb)
 {
-	int loopvar = 1000;
+	int loopvar = 2000;
 
 	qbman_swp_mc_submit(swp, cmd, cmd_verb);
 
-- 
2.17.1

